cmd/mt-whisper-importer-reader/main.go

```diff
@@ -76,48 +65,34 @@ var (
 		"/opt/graphite/storage/whisper",
 		"The directory that contains the whisper file structure",
 	)
-	readArchivesStr = flag.String(
-		"read-archives",
-		"*",
-		"Comma separated list of positive integers or '*' for all archives",
-	)
 	httpAuth = flag.String(
 		"http-auth",
 		"",
 		"The credentials used to authenticate in the format \"user:password\"",
 	)
-	chunkSpans   []uint32
-	readArchives map[int]struct{}
-	printLock    sync.Mutex
+	dstSchemas = flag.String(
+		"dst-schemas",
+		"",
+		"The filename of the output schemas definition file",
+	)
+	nameFilterPattern = flag.String(
+		"name-filter",
+		"",
+		"A regex pattern to be applied to all metric names, only matching ones will be imported",
+	)
+	printLock  sync.Mutex
```

Comment:
printLock and exitOnError are no longer used.  btw "exit on error" is still the behavior right, with this log library switch?